R/utils.R

```diff
@@ -35,3 +35,25 @@ prep_info <- function(input) {
     str_trim() #%>%
   # str_replace_all("[[:punct:]]", "")
 }
+
+
+#' Function which produces a vector to be used in string cleaning from scraping there are a lot of hard coded values in here and will need to be adapted for the weird nuances.
+cleaning <- function() {
```

Comment:
this creates a vector which is the main source of plain text cleaning. It will require more hardcoding as issues arise.